independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/QuarkusCommandHandlers.java

```diff
@@ -129,16 +130,16 @@ static SelectionResult select(final String query, final Collection<Extension> al
             listedExtensions.stream()
                     .filter(extension -> pattern.matcher(extension.getName().toLowerCase()).matches()
                             || pattern.matcher(extension.getArtifact().getArtifactId().toLowerCase()).matches()
-                            || pattern.matcher(extension.getShortName().toLowerCase()).matches()
-                            || matchLabels(pattern, extension.getKeywords()))
+                            || pattern.matcher(ExtensionProcessor.of(extension).getShortName().toLowerCase()).matches()
+                            || matchLabels(pattern, ExtensionProcessor.of(extension).getExtendedKeywords()))
```

Comment:
I see you are using `ExtensionProcessor.of(extension)` all over the place just to call a single getter.
Instead of creating new instances of `ExtensionProcessor` for the same extension instance, it'd be more efficient to have methods e.g. `ExtensionProcessor.getExtendedKeywords(extension)`, you could still have instance methods e.g. `ExtensionProcessor.getExtendedKeywords()` call the corresponding static one. WDYT?

Reply:
@aloubyansky I did a hard replace, I should have a look if I can refactor with a variable for cases like above. I will fix that one.

I also agree with what you said and I am going to see to converts the ones that are often called alone.

Reply:
Ok you were 100% right and this is now fixed :) Good one!